src/main/java/org/mskcc/oncokb/curation/importer/CoreImporter.java

```diff
@@ -45,11 +48,15 @@ public CoreImporter(
         this.drugService = drugService;
         this.nciThesaurusService = nciThesaurusService;
         this.mainService = mainService;
+        this.applicationProperties = applicationProperties;
+
+        DATA_DIRECTORY = applicationProperties.getOncokbDataRepoPath() + "/curation/oncokb/";
```

Comment:
Not sure if we want to keep consistent, but we had a naming convention where DIRECTORY is the path without the slash and DIR_PATH is with slash